libs/data-access/gn4/src/openapi/api/records.api.service.ts

```diff
@@ -23,7 +23,10 @@ import {
 import { CustomHttpParameterCodec } from '../encoder'
 import { Observable } from 'rxjs'
 
-import { BatchEditParameterApiModel } from '../model/models'
+import {
+  AssociatedRecordApiModel,
+  BatchEditParameterApiModel,
+} from '../model/models'
```

Comment:
why is that change necessary? Having manual changes in there is going to bite us later if we keep updating the client automatically.